### DISCUSSION TITLE
fix for iOS devices patch on Railway

### DISCUSSION THREAD

**Discussion by ToasterSticks** [coder/code-server#3068] [Q&A] [answered]
As mentioned in #2975, there was a patch to fix support for iOS devices. How could I get this to work on railway, if it is even possible?

**Comment by oxy** [comment, marked as answer]
You should be able to do it by just adding the `sed` commands to your Dockerfile - see https://github.com/cdr/deploy-code-server/blob/main/guides/railway.md#step-3-modify-your-environment-to-add-additional-tools

**Reply by ToasterSticks** [reply]
Do you know which line(s) I should put them between?

**Reply by bpmct** [reply]
You can do this around like 25. See this guide for some help writing Dockerfiles: https://docs.docker.com/engine/reference/builder/#environment-replacement

**Reply by jsjoeio** [reply]
The patch should be in 3.9.3 if you want to upgrade instead of having to apply the patch.
